core/src/main/resources/db/cgds.sql

```diff
@@ -695,4 +713,4 @@ CREATE TABLE `info` (
   `DB_SCHEMA_VERSION` varchar(8)
 ) DEFAULT CHARSET=utf8;
 -- THIS MUST BE KEPT IN SYNC WITH db.version PROPERTY IN pom.xml
-INSERT INTO info VALUES ('1.3.0');
+INSERT INTO info VALUES ('1.3.2');
```

Comment:
I agree, it will help older portals to run on a newer DB (just wondering if this will happen often). I will update the number.